configure.ac

```diff
@@ -1182,6 +1182,23 @@ AS_IF([test "$USE_NLS" = "yes"], [
   AC_MSG_NOTICE([Language support with gettext not available])
 ]) 
 
+#############################################################
+# Sort out fmt
+#############################################################
+
+dnl If in a git repo, get submodule, unless BOUT_DONT_UPDATE_GIT_SUBMODULE is set
+AC_CHECK_FILE([.git], [
+  AS_IF([test "x$BOUT_DONT_UPDATE_GIT_SUBMODULE" == "x"], [
+    git submodule update --init externalpackages/fmt
+  ])
+])
+
+dnl Copy the one file we need to somewhere else
+AC_CONFIG_LINKS(src/fmt/format.cxx:externalpackages/fmt/src/format.cc)
```

Comment:
It would be nice if we would support compiling with the fmt library that is already installed on the system.

Reply:
We do, using CMake. Unfortunately, it was very difficult to do so with the makefile system, so I gave up.